packages/css/src/components/select/select.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@mixin reset {
+  appearance: none;
+  border: none;
+}
+
+.ams-select {
+  background-color: var(--ams-select-background-color);
+  box-shadow: var(--ams-select-box-shadow);
+  color: var(--ams-select-color);
+  font-family: var(--ams-select-font-family);
+  font-size: var(--ams-select-font-size);
+  font-weight: var(--ams-select-font-weight);
+  line-height: var(--ams-select-line-height);
+  max-inline-size: var(--ams-select-max-inline-size);
```

Comment:
Yes, sizing inputs is of later concern.

Reply:
I think we still need `max-inline-size: 100%` to prevent the component from breaking out of its container

Reply:
The reason I've created a token is because NLDS seems to set this and perhaps you're like to be able to override this in your application somehow.

Reply:
I don't think we should. The Utrecht Select combines `max-inline-size` with `inline-size: 100%` to make sure the select doesn't break out of its container. Seeing as we don't do that, allowing users to set `max-inline-size` to a static value would break the component.

Reply:
I've added `inline-size: 100%`. Preventing users to set a width seems like a bad idea.

Reply:
People can still add a class name and set a width. We haven’t researched how to size inputs yet. On the grid? Through a set of t-shirt sizes? A number of variants? And how to implement variations for varying screen widths. So 100% is a good, predictable start.

Reply:
So it's ok as is?

Reply:
I don't realy get the argument to not use a token but here you go.